_includes/subscribe.html

```diff
@@ -1,7 +1,7 @@
 {% if page.title == 'Meshery | The Kubernetes and Cloud Native Manager' %}
 
 <style>
-  @media (min-width: 841px) and (max-width: 1054px) {
+  @media (min-width: 841px) and (max-width: 1210px) {
```

Comment:
make it 1229px  ,a minor but still it caught my eye. 
![image](https://user-images.githubusercontent.com/95926324/194038161-bf26aa2c-95d9-417c-81fe-b81772e02538.png)